test_vm/src/lib.rs

```diff
@@ -116,15 +151,117 @@ pub struct NetworkStats {
     pub total_client_storage_fee: TokenAmount,
 }
 
+// accounts for verifreg root signer and msig
 pub const VERIFREG_ROOT_KEY: &[u8] = &[200; fvm_shared::address::BLS_PUB_LEN];
 pub const TEST_VERIFREG_ROOT_SIGNER_ADDR: Address = Address::new_id(FIRST_NON_SINGLETON_ADDR);
 pub const TEST_VERIFREG_ROOT_ADDR: Address = Address::new_id(FIRST_NON_SINGLETON_ADDR + 1);
-// Account actor seeding funds created by new_with_singletons
+// account actor seeding funds created by new_with_singletons
 pub const FAUCET_ROOT_KEY: &[u8] = &[153; fvm_shared::address::BLS_PUB_LEN];
 pub const TEST_FAUCET_ADDR: Address = Address::new_id(FIRST_NON_SINGLETON_ADDR + 2);
 pub const FIRST_TEST_USER_ADDR: ActorID = FIRST_NON_SINGLETON_ADDR + 3;
 
-// accounts for verifreg root signer and msig
+impl<'bs, BS> VM<BS> for TestVM<'bs, BS>
+where
+    BS: Blockstore,
```

Comment:
Eventually will move `TestVM` to a separate submodule, but I've left it in this file so the diff is clear